### PR TITLE
#165762516 Integrate react-toolkit

### PR DESCRIPTION
**Pull request by dannylwe**
#### What does this PR do?
Integrate react-toolkit

#### Description of Task to be completed?
Get reusable components from react-toolkit

#### How should this be manually tested?
fetch `git fetch ch-intergrate-react-toolkit-165762516`
checkout `git checkout ch-intergrate-react-toolkit-165762516`
run `npm install`
run `npm run`

#### What are the relevant pivotal tracker stories?
[#165762516](https://www.pivotaltracker.com/story/show/165762516)

#### Screenshots (if appropriate):
Screenshot 2019-05-02 at 12 00 53

### CODE REVIEW

File: src/components/Btn.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Button } from 'react-toolbox/lib/button';
+
+const Btn = () => (
```
Comment (adesege) [inline src/components/Btn.js]: Change the name of the component to `Button` instead

File: jest.config.js [hunk 2]
```diff
@@ -39,5 +42,7 @@ module.exports = {
 
   // Indicates whether each individual test should be reported during the run
   verbose: false,
-
+  moduleNameMapper: {
+    "\\.(css|scss|jpg|png)$": "<rootDir>/ignore-css.js"
```
Comment (adesege) [inline jest.config.js:46]: I left a comment regarding the name of this file.

File: package.json [hunk 3]
```diff
@@ -23,10 +23,16 @@
   "homepage": "https://github.com/andela/ah-frontend-sparta#readme",
   "dependencies": {
     "babel-jest": "^24.7.1",
+    "css-loader": "^2.1.1",
     "enzyme": "^3.9.0",
     "enzyme-adapter-react-16": "^1.12.1",
+    "postcss": "^7.0.14",
+    "postcss-calc": "^7.0.1",
```
Comment (adesege) [inline package.json]: It doesn't appear that you are using any of these postcss packages except postcss-loader. Remove extraneous packages

**Review by adesege — CHANGES_REQUESTED** [review]

**Comment by adesege** [comment]
Rebase with develop

File: package.json (hunk 3 above)
Comment (dannylwe) [reply package.json]: @adesege worked on feedback

**Comment by adesege** [comment]
@dannylwe Rebase this branch with develop

**Comment by adesege** [comment]
@dannylwe You are yet to fix merge conflict on this PR

**Pull request closed without merging**